Compile install failed: make[2]: *** [CMakeFiles/duckdb_proj.dir/build.make:86: extra/duckdb/build/release/stamp/duckdb_proj-build] Error 2

### My Env:
PLATFORM_ID="platform:el8"
PRETTY_NAME="Oracle Linux Server 8.7"
gcc (GCC) 8.5.0 20210514 (Red Hat 8.5.0-28.0.1)
ldd (GNU libc) 2.28

### Build from source but Failed many times:
cmd: -- Specify dir
sh build.sh -t release -d /u01/data/alisql_src/

### Some error:
/u01/data/scripts/gilbert/AliSQL/mysys/hash.cc: In function ‘my_hash_value_type my_hash_sort(CHARSET_INFO*, const uchar*, size_t)’:
/u01/data/scripts/gilbert/AliSQL/mysys/hash.cc:48:32: warning: cast from type ‘const uchar*’ {aka ‘const unsigned char*’} to type ‘uchar
*’ {aka ‘unsigned char*’} casts away qualifiers [-Wcast-qual]
   my_ci_hash_sort(cs, (uchar *)key, length, &nr1, &nr2);

### Detail log:
[compile_err.log](https://github.com/user-attachments/files/26265640/compile_err.log)

Please help check my error log, thanks.

--- a/wiki/duckdb/how-to-setup-duckdb-node-en.md
+++ b/wiki/duckdb/how-to-setup-duckdb-node-en.md
@@ -225,8 +225,8 @@ duckdb_mode=ON
 duckdb_require_primary_key=ON
 force_innodb_to_duckdb=ON
 
-dml_in_batch=ON
-update_modified_column_only=ON
+duckdb_dml_in_batch=ON
+duckdb_update_modified_column_only=ON
 duckdb_multi_trx_in_batch=ON
 duckdb_multi_trx_timeout=5000
 duckdb_multi_trx_max_batch_length=268435456

--- a/wiki/duckdb/how-to-setup-duckdb-node-zh.md
+++ b/wiki/duckdb/how-to-setup-duckdb-node-zh.md
@@ -238,8 +238,8 @@ duckdb_mode=ON
 duckdb_require_primary_key=ON
 force_innodb_to_duckdb=ON
 
-dml_in_batch=ON
-update_modified_column_only=ON
+duckdb_dml_in_batch=ON
+duckdb_update_modified_column_only=ON
 duckdb_multi_trx_in_batch=ON
 duckdb_multi_trx_timeout=5000
 duckdb_multi_trx_max_batch_length=268435456

